File: src/defaults.py
# Default values for frame design (all measurements in inches)

# Artwork dimensions
DEFAULT_ARTWORK_HEIGHT = 12.5
DEFAULT_ARTWORK_WIDTH = 18.75

# Mat settings
DEFAULT_INCLUDE_MAT = True
DEFAULT_MAT_WIDTH = 2.0
DEFAULT_MAT_OVERLAP = 0.125

# Frame dimensions
DEFAULT_FRAME_MATERIAL_WIDTH = 0.75  # Width of frame molding (face)
DEFAULT_FRAME_THICKNESS = 0.75       # Depth of frame stock (z-axis)
DEFAULT_RABBET_DEPTH = 0.375        # Rabbet extension (x/y plane)

# Material thicknesses (z-axis)
DEFAULT_GLAZING_THICKNESS = 0.093   # ~3/32" glass/acrylic
DEFAULT_MATBOARD_THICKNESS = 0.055  # ~1/16" 4-ply matboard
DEFAULT_ARTWORK_THICKNESS = 0.008   # Photo paper / thin print
DEFAULT_BACKING_THICKNESS = 0.125   # 1/8" foam core or hardboard

# Cutting/tool settings
DEFAULT_BLADE_WIDTH = 0.125         # 1/8" saw blade kerf

# Standard artwork size keys for consistent referencing
STANDARD_ARTWORK_SIZES = {
    "8x10": {"height": 8.0, "width": 10.0, "name": "8×10"},
    "10x15": {"height": 10.0, "width": 15.0, "name": "10×15"},
    "11x14": {"height": 11.0, "width": 14.0, "name": "11×14"},
    "16x20": {"height": 16.0, "width": 20.0, "name": "16×20"},
}

# Default starting artwork size key (can be referenced throughout the app)
DEFAULT_ARTWORK_SIZE_KEY = "10x15"

File: core/src/defaults.rs
// Default values for frame design (all measurements in inches)

// Artwork dimensions
pub const DEFAULT_ARTWORK_HEIGHT: f64 = 12.5;
pub const DEFAULT_ARTWORK_WIDTH: f64 = 18.75;

// Mat settings
pub const DEFAULT_INCLUDE_MAT: bool = true;
pub const DEFAULT_MAT_WIDTH: f64 = 2.0;
pub const DEFAULT_MAT_OVERLAP: f64 = 0.125;

// Frame dimensions
pub const DEFAULT_FRAME_MATERIAL_WIDTH: f64 = 0.75;  // Width of frame molding (face)
pub const DEFAULT_FRAME_THICKNESS: f64 = 0.75;        // Depth of frame stock (z-axis)
pub const DEFAULT_RABBET_DEPTH: f64 = 0.375;         // Rabbet extension (x/y plane)

// Material thicknesses (z-axis)
pub const DEFAULT_GLAZING_THICKNESS: f64 = 0.093;    // ~3/32" glass/acrylic
pub const DEFAULT_MATBOARD_THICKNESS: f64 = 0.055;   // ~1/16" 4-ply matboard
pub const DEFAULT_ARTWORK_THICKNESS: f64 = 0.008;    // Photo paper / thin print
pub const DEFAULT_BACKING_THICKNESS: f64 = 0.125;    // 1/8" foam core or hardboard

// Assembly
pub const DEFAULT_ASSEMBLY_MARGIN: f64 = 0.0625;     // 1/16" clearance for assembly

// Cutting/tool settings
pub const DEFAULT_BLADE_WIDTH: f64 = 0.125;          // 1/8" saw blade kerf
